src/member.ts

```diff
@@ -219,6 +221,7 @@ export class Member implements IStateful<IMemberState> {
         return {
             address: item.address,
             contract: item.contract,
+            createdAt: Number(item.createdAt),
```

Comment:
Just curious:  Why not return a Date?

Reply:
Date is an alias for number in this library, we use numbers for dates right now